### ISSUE TITLE
第32回(平日回) - イベントの内容・日時を決める

### ISSUE THREAD

**Issue by PharaohKJ** [jaws-ug-kanazawa/jaws-ug-kanazawa.github.io#113]
- [x] 日時
  - [x] Google カレンダーに記入
- [x] 内容
- [x] テーマ
- [x] セッションタイトル1
  - [x] スピーカ
  - [x] タイトル
  - [x] doorkeeper への掲載
- [ ] セッションタイトル2
  - [ ] スピーカ
  - [ ] タイトル
  - [ ] doorkeeper への掲載
- [ ] セッションタイトル3
  - [ ] スピーカ
  - [ ] タイトル
  - [ ] doorkeeper への掲載
- [ ] セッションタイトル4
  - [ ] スピーカ
  - [ ] タイトル
  - [ ] doorkeeper への掲載

**Comment by PharaohKJ** [comment]
2018-04-13（金）19:00 - 21:00
JAWS DAYS 報告会

セッション1 @PharaohKJ

**Comment by PharaohKJ** [comment]
他枠について JAWS-UG 本家 slack 中部チャンネルで募集しました。
あとAWSJからゲストがいらっしゃる予定、そちらの対応もタスクにいれておくこと。

**Comment by aibax** [comment]
終了
